ifupdown2/addons/batman_adv.py

```diff
@@ -84,224 +85,223 @@ class batman_adv(Addon, moduleBase):
 
         # Verify existance of batman interfaces (should be present already)
         batman_ifaces = []
-        for iface in self._get_batman_ifaces (ifaceobj):
-            if not self.cache.link_exists (iface):
-                self.logger.warn ('batman iface %s not present' % iface)
+        for iface in self._get_batman_ifaces(ifaceobj):
+            if not self.cache.link_exists(iface):
+                self.logger.warn('batman iface %s not present' % iface)
                 continue
 
-            batman_ifaces.append (iface)
+            batman_ifaces.append(iface)
 
-        if len (batman_ifaces) == 0:
-            raise Exception ("None of the configured batman interfaces are available!")
+        if len(batman_ifaces) == 0:
+            raise Exception("None of the configured batman interfaces are available!")
 
-        routing_algo = ifaceobj.get_attr_value_first ('batman-routing-algo')
+        routing_algo = ifaceobj.get_attr_value_first('batman-routing-algo')
         if routing_algo:
-            self._set_routing_algo (routing_algo)
+            self._set_routing_algo(routing_algo)
 
-        if_ignore_re = self._get_batman_ifaces_ignore_regex (ifaceobj)
+        if_ignore_re = self._get_batman_ifaces_ignore_regex(ifaceobj)
         # Is the batman main interface already present?
-        if self.cache.link_exists (ifaceobj.name):
+        if self.cache.link_exists(ifaceobj.name):
             # Verify which member interfaces are present
-            members = self._find_member_ifaces (ifaceobj)
+            members = self._find_member_ifaces(ifaceobj)
             for iface in members:
                 if iface not in batman_ifaces:
-                    self._batctl_if (ifaceobj.name, iface, 'del')
+                    self._batctl_if(ifaceobj.name, iface, 'del')
             for iface in batman_ifaces:
                 if iface not in members:
-                    self._batctl_if (ifaceobj.name, iface, 'add')
+                    self._batctl_if(ifaceobj.name, iface, 'add')
 
         # Batman interfaces no present, add member interfaces to create it
         else:
             for iface in batman_ifaces:
-                self._batctl_if (ifaceobj.name, iface, 'add')
+                self._batctl_if(ifaceobj.name, iface, 'add')
 
         # Check/set any B.A.T.M.A.N. adv. set within interface configuration
         for attr in self._batman_attrs:
-            value_cfg = self._get_batman_attr (ifaceobj, attr)
-            if value_cfg and value_cfg != self._read_current_batman_attr (ifaceobj, attr):
-                self._set_batman_attr (ifaceobj, attr, value_cfg)
+            value_cfg = self._get_batman_attr(ifaceobj, attr)
+            if value_cfg and value_cfg != self._read_current_batman_attr(ifaceobj, attr):
+                self._set_batman_attr(ifaceobj, attr, value_cfg)
 
         if ifaceobj.addr_method == 'manual':
+            netlink = NetlinkManager()
```

Comment:
So far this is the only "real" change - I assume this is correct sicne it is the only class exposing the function used.

Reply:
the change should be:
`self.netlink.link_up(ifaceobj.name)`